projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/store/isa-data.ts

```diff
@@ -123,17 +126,13 @@ export const isaFilteredImages = derived(
  */
 export type ISA_Data = z.infer< typeof ImageData >;
 export const isaData = image_size_analysis.store;
-export const isaDataLoading = image_size_analysis.pending;
+export const isaDataLoading = derived(
+	[ image_size_analysis.pending, isaData ],
+	( [ $pending, $data ] ) => {
+		// Special case for ignored since that's currently handled by a different store.
+		// Ultimately, I think "ignored" is going back to the main store.
+		// because it too is stored over the network.
```

Comment:
Yeah, I think moving ignored into this store is starting to make more and more sense.